### PR TITLE
Fetch rocket api

### LINKED ISSUE #17: [4pt]  Fetch rockets - Fetch data
Fetch data from the Rockets endpoint  (https://api.spacexdata.com/v3/rockets) when the application starts (as Rockets is the default view).

Once the data are fetched, dispatch an action to store the selected data in Redux store:

- id
- name
- type
- flickr_images

NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).

### PR DESCRIPTION
**Pull request by SimonGideon**
In this PR I:
- Fetch data from the Rockets endpoint (https://api.spacexdata.com/v3/rockets) when the application starts (as Rockets is the default view).
- Once the data are fetched, dispatch an action to store the selected data in Redux store:
              - id
             - name
             - type
             - flickr_images

:slightly_smiling_face: Thanks for the review

### CODE REVIEW

File: src/components/Rockets.js [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+  }, [dispatch, rockets.length]);
+
+  return (
+    <div>
+      {rockets.map((e) => (
+        <div key={e.rocket_id}>
+          <div>{e.rocket_id}</div>
+          <div>{e.rocket_name}</div>
+          <div>{e.description}</div>
+          <div>{e.images}</div>
```
Comment (sebalp100) [inline src/components/Rockets.js:22]: Kindly fix your code so the images display correctly, that way your project is going to look more professional. You can try using an IMG element and the data as the source.

**Review by sebalp100 — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED ♻️** 

Hi, @SimonGideon   good job so far following the instructions. However, there are still some requirements missing from your project. You can check the comments at the bottom for more info.

dog


**HIGHLIGHTS:**

- Good job using the correct ES6 syntax 💯 
- Branch name is descriptive 💯 
- Commits are descriptive 💯 
- Linters are passing all the tests 💯
- Data if fetching correctly

**REQUIRED CHANGES ♻️  :**

Great job so far, however, there are still some changes to be made before you can continue.

_For more information on your required changes, please check below ⬇️_

File: src/components/Rockets.js (hunk 1 above)
Comment (SimonGideon) [reply src/components/Rockets.js:22]: @sebalp100 This is implemented in the display section. The core requirement is to **fetch the data**. I would suggest checking the requested changes on the feature commit.

Comment (sebalp100) [reply src/components/Rockets.js:22]: @SimonGideon You are correct. Using the IMG element and the data as the source should still work for your next card. Send another request and I can approve it 👍

Comment (SimonGideon) [reply src/components/Rockets.js:22]: @sebalp100 Sure.

**Review by sebalp100 — APPROVED** [review]
STATUS: APPROVED 🟢

ok


Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name
✔️ Functionality working as expected

Great implementation of all the requirements in this milestone. 👏

You have done an amazing job 👍

Your project is complete! There is nothing else to say other than... it's time to merge it. :shipit:

Congratulations! 🎉

CHEERS AND HAPPY CODING!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by SimonGideon** [comment]
@sebalp100 Thanks for the review

**Pull request merged**